src/ExpressionParser.php

```diff
@@ -864,10 +629,13 @@ private function setDeprecationCheck(bool $deprecationCheck): bool
         return $current;
     }
 
-    private function createArguments(int $line): ArrayExpression
+    /**
+     * @internal
+     */
+    public function createCallableArguments(int $line, bool $parseOpenParenthesis = true): ArrayExpression
```

Comment:
The name `create*` is weird to me. This method is parsing callable arguments, so I would name it `parse*` for consistency with other methods

Reply:
Updated